bool reached_goal
---
nav_msgs/Path path